### DISCUSSION TITLE
Can a community have a special profile repo ??

### DISCUSSION THREAD

**Discussion by AakarshTeja** [EddieHubCommunity/support#262] [Q&A] [answered]
Recently Github got a feature of creating a special Repo that is same as the username and its readme.md will appear on the user's profile. Does that feature works for a community ??
@eddiejaoude  Can we Try that once ??

**Comment by eddiejaoude** [comment, marked as answer]
That feature is not available yet. Good idea to try 👍 

 I did just try, just in case it was a hidden feature, but unfortunately it didnt work ....

<img width="834" alt="Screenshot 2020-09-29 at 00 18 31" src="https://user-images.githubusercontent.com/624760/94495345-621ce280-01e9-11eb-983b-95a94914ba2b.png">

**Reply by AakarshTeja** [reply]
It would be very nice if we had that feature for communities too.

**Reply by eddiejaoude** [reply]
Yes I agree 👍 - I am sure it will come soon 🤓
